packages/sdk/tests/unit/dataProtectorCore/getGrantedAccess.unit.ts

```diff
@@ -0,0 +1,265 @@
+import { describe, expect, it, jest } from '@jest/globals';
+import { ApiCallError } from 'iexec/errors';
+import { ValidationError } from 'yup';
+import { WorkflowError } from '../../../src/index.js';
+import { getGrantedAccess } from '../../../src/lib/dataProtectorCore/getGrantedAccess.js';
+
+describe('getGrantedAccess', () => {
+  describe('Check validation for input parameters', () => {
+    describe('When given protected data is invalid', () => {
```

Comment:
multiple `describe` blocks are used here, but many tests share the same configuration and context
we can use a single `describe` block for related tests to make tests suite easier to understand

Reply:
> but many tests share the same configuration and context

What do you mean by "configuration and context"?
What piece of code could be extracted and moved to the top of the file?